File: Edinburgh College/JS/2. Conditional/activity1.js
let firstNumber = Number(prompt("First number: "))
let secondNumber = Number(prompt("Second number: "))
let thirdNumber = Number(prompt("Third number: "))

if (firstNumber == secondNumber && firstNumber == thirdNumber) {
    console.log("All numbers are equal")
} else if ((firstNumber == secondNumber && firstNumber != thirdNumber) || (firstNumber != secondNumber && firstNumber == thirdNumber)) {
    console.log("Neither all are equal or different")
} else {
    console.log("All numbers are different")
}

File: Edinburgh College/Python/2. Python Control Statements/activity1.py
# Control statements

# Get first number
first_number = int(input("Enter first number: "))

# Get second number
second_number = int(input("Enter second number: "))

# Get third number
third_number = int(input("Enter third number: "))

if first_number == second_number and first_number == third_number:
    print("All numbers are equal.")
elif (first_number != second_number and first_number != third_number and
      second_number != third_number):
    print("All numbers are different.")
else:
    print("Neither all are equal or different.")